core/trino-main/src/test/java/io/trino/execution/TestDropTableTask.java

```diff
@@ -63,6 +63,24 @@ public void testDropNotExistingTableIfExists()
         // no exception
     }
 
+    @Test
+    public void testDropNotExistingSchemaIfExists()
+    {
+        QualifiedName tableName = QualifiedName.of(TEST_CATALOG_NAME, "non_existing_schema", "not_existing_table");
+
+        getFutureValue(executeDropTable(tableName, true));
+        // no exception
+    }
+
+    @Test
+    public void testDropNotExistingCatalogIfExists()
+    {
+        QualifiedName tableName = QualifiedName.of("non_existing_catalog", "non_existing_schema", "not_existing_table");
+
+        getFutureValue(executeDropTable(tableName, true));
+        // no exception
+    }
```

Comment:
`testDropNotExistingCatalogIfExists` → `testDropNotExistingSchemaIfExists` → `testDropNotExistingTableIfExists` is the natural order to me.